Remotion/Development/UnitTesting.IsolatedCodeRunner/IsolatedCodeRunner.cs

```diff
@@ -0,0 +1,167 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.ExceptionServices;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Remotion.Utilities;
+
+namespace Remotion.Development.UnitTesting.IsolatedCodeRunner
+{
+  public class IsolatedCodeRunner
```

Comment:
we should seal this type (and the others in this assembly, too). I know, in the past I rarely sealed and I mostly wet to sealing here because of the Run method's assembly lookup. still feels like a good idea, though.